### PR TITLE
Nerf sell value of some common items

### PR DESCRIPTION
**Pull request by KittyTac**
#### Summary
Balance "Nerf sell value of adhesive bandages, water, and cotton patches"

#### Purpose of change
Adhesive bandages are nearly useless for stopping bleeding, (clean) water is easy to get by the hundreds of units by leaving a water tank on a fire, and cotton patches can be easily gotten in copious amounts by cutting up clothes. They were all overpriced.

#### Describe the solution
Change the adhesive bandage value from 2 dollars to 2 cents, water value from 50 cents to 1 cent, and cotton patch value from 50 cents to 2 cents.

#### Describe alternatives you've considered
Leaving as is, or having different numbers.

#### Testing
None needed.

#### Additional context
N/A.

### CODE REVIEW

File: data/json/items/comestibles/drink.json [hunk 1]
```diff
@@ -1331,6 +1331,7 @@
     "weight": "250 g",
     "volume": "250 ml",
     "price": 50,
+    "price_postapoc": 5,
```
Comment (Maleclypse) [inline data/json/items/comestibles/drink.json:1334]:
```suggestion
    "price_postapoc": 1,
```

File: data/json/items/tool/toiletries.json [hunk 2]
```diff
@@ -160,7 +160,7 @@
     "volume": "6 ml",
     "//": "This is calculated from medium weight cotton at 203 GSM, with a size of 6 inches square",
     "price": 200,
-    "price_postapoc": 50,
+    "price_postapoc": 5,
```
Comment (Maleclypse) [inline data/json/items/tool/toiletries.json:163]:
```suggestion
    "price_postapoc": 2,
```
Aren't these mass produced by butchering clothes?  If they take more work than that 5 cents is probably ok.

Comment (KittyTac) [reply data/json/items/tool/toiletries.json]: Yeah, they're mass-produced. I didn't want to overnerf but this seems even better on a second thought.

**Comment by Cimanyd0** [comment]
Related to #59129

**Comment by PatrikLundell** [comment]
I disagree with the "no testing needed" assessment. Making the appropriate changes to the wrong file will cause the desired effects not to apply (like changing the game you currently play, as opposed to changing the branch you intended to change).
I'd probably check one item from each file changed (unless you, unlike me, never makes any mistakes).

**Comment by RelMayers** [comment]
Testing always should be done.

**Review by Maleclypse — APPROVED** [review]

**Pull request merged**